blobfuse2-nightly.yaml

```diff
@@ -48,7 +42,7 @@ parameters:
     default: false
 
 jobs:
-  # Ubuntu based test suite
+  # Ubuntu Tests
   - job: Set_1
```

Comment:
We need to add Ubn-22 here.

Reply:
For Debb-9 we can add one more set and have our self-hosted agent for now? or we want to move completely to 1ES and see it later?

Reply:
yes I can add that

Reply:
Done